platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -0,0 +1,247 @@
+package com.mapbox.mapboxsdk.offline;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * For the present, a tile pyramid is the only type of offline region. In the future,
```

Comment:
Good catch, I updated the docs in https://github.com/mapbox/mapbox-gl-native/commit/a15175c46e98c4d2cdf53c94180ab0f6edc2e1d3.